File: app/src/main/java/com/alvinhkh/buseta/model/ArrivalTime.java
package com.alvinhkh.buseta.model;


public class ArrivalTime {

    public Integer capacity = -1;

    public String companyCode = "";

    public String estimate = "";

    public String expire = "";

    public Boolean expired = false;

    public String id = "0";

    public Boolean isSchedule = false;

    public Boolean hasWheelchair = false;

    public Boolean hasWifi = false;

    public String text = "";

    public String isoTime = "";

    public Float distanceKM = -1.0f;

    public String plate = "";

    public Double latitude = 0.0;

    public Double longitude = 0.0;

    public String platform = "";

    public String destination = "";

    public String direction = "";

    public Long generatedAt = 0L;

    public Long updatedAt = 0L;

    public ArrivalTime() { }

    public String toString() {
        return "ArrivalTime{capacity=" + this.capacity + ", companyCode=" + this.companyCode
                + ", destination=" + this.destination + ", direction=" + this.direction + ", distanceKM=" + this.distanceKM
                + ", estimate=" + this.estimate + ", expire=" + this.expire + ", expired=" + this.expired
                + ", id=" + this.id + ", isoTime=" + this.isoTime + ", isSchedule=" + this.isSchedule
                + ", hasWheelchair=" + this.hasWheelchair + ", hasWifi=" + this.hasWifi
                + ", text=" + this.text + ", plate=" + this.plate + ", platform=" + this.platform
                + ", latitude=" + this.latitude + ", longitude=" + this.longitude
                + ", generatedAt=" + this.generatedAt + ", updatedAt=" + this.updatedAt + "}";
    }
}

File: app/src/main/java/com/alvinhkh/buseta/model/ArrivalTime.kt
package com.alvinhkh.buseta.model


data class ArrivalTime (
    var capacity: Int = -1,
    var companyCode: String = "",
    var estimate: String = "",
    var expire: String = "",
    var expired: Boolean = false,
    var id: String = "0",
    var isSchedule: Boolean = false,
    var hasWheelchair: Boolean = false,
    var hasWifi: Boolean = false,
    var text: String = "",
    var isoTime: String = "",
    var distanceKM: Float = -1.0f,
    var plate: String = "",
    var latitude: Double = 0.0,
    var longitude: Double = 0.0,
    var platform: String = "",
    var destination: String = "",
    var direction: String = "",
    var generatedAt: Long = 0L,
    var updatedAt: Long = 0L,
    var note: String = ""
)
